.github/workflows/stale-pr.yml

```diff
@@ -12,5 +12,7 @@ jobs:
         with:
           start-date: '2024-04-25T00:00:00Z'
```

Comment:
We should remove the start date now as well, this was added as a grace period when the stale bot was added but no longer needed.